src/ape/api/providers.py

```diff
@@ -166,3 +166,17 @@ def send_transaction(self, txn: TransactionAPI) -> ReceiptAPI:
     @abstractmethod
     def get_events(self, **filter_params) -> Iterator[dict]:
         ...
+
+
+class TestProviderAPI(ProviderAPI):
```

Comment:
As part of the test work, we filter out accounts that implement this API and `ape test` _only_ gets accounts that implement this API.